### ISSUE TITLE
Allow using VMTK functions from a C++ Slicer module

### ISSUE THREAD

**Issue by cpinter** [vmtk/SlicerExtension-VMTK#69]
Currently it does not seem possible to use VMTK functions from a C++ loadable Slicer module. It would be needed to expose the `*_INCLUDE_DIRS` and `*_LIBRARY_DIRS` CMake variables in `SlicerVMTKConfig.cmake`.

For an example see how the [SlicerCollaboration](https://github.com/EBATINCA/SlicerCollaboration) extension uses the [SlicerOpenIGTLink](https://github.com/openigtlink/SlicerOpenIGTLink) extension (that is also a superbuild extension just like SlicerVMTK).

**Comment by lassoan** [comment]
Good idea. If you need any of these then someone could add these and send a pull request.
